Doc/library/collections.rst

```diff
@@ -698,9 +698,9 @@ stack manipulations such as ``dup``, ``drop``, ``swap``, ``over``, ``pick``,
 :class:`defaultdict` objects
 ----------------------------
 
-.. class:: defaultdict([default_factory[, ...]])
+.. class:: defaultdict(default_factory=None, /, [...])
 
-    Returns a new dictionary-like object.  :class:`defaultdict` is a subclass of the
+    Return a new dictionary-like object.  :class:`defaultdict` is a subclass of the
```

Comment:
`Returns` is good. https://google.github.io/styleguide/pyguide.html#383-functions-and-methods

The description could provide a high level explanation what is so specific about returned dictionary.

Reply:
The active rather than passive preferred. `https://www.python.org/dev/peps/pep-0257/`

Reply:
`Returns` is active, no?

Reply:
Sorry, I mistated the constrast, but the PEP is clear.

Reply:
Unlike Google Style Guide, the PEP fails to explain the rationale.